Dockerfile.bullseye

```diff
@@ -0,0 +1,132 @@
+FROM php:8.2.0RC4-zts-bullseye AS php-base
```

Comment:
Can we delete this file if it is no longer used?

Reply:
yes, I'll remove it